Add issue templates (#68)
hexatomic/hexatomic: .github/ISSUE_TEMPLATE/feature_request.md

---
name: Feature request

> about: Suggest an idea for this project

Reviewer: ```suggestion
about: Suggest an idea for Hexatomic, e.g., new functionality.
```

Reviewer: Be more specific what we mean with idea, and brand the about.

title: ''
labels: 'enhancement'
assignees: ''

---

**Is your feature request related to a problem? Please describe.**

> A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]

Reviewer: ```suggestion
A clear and concise description of what the problem is. Example: "I need Hexatomic to do [...]"
```

Be more positive. We don't expect our users to be frustrated :).
